### DISCUSSION TITLE
PWM off TCD on 14-pin DD series

### DISCUSSION THREAD

**Discussion by fijam** [SpenceKonde/DxCore#538] [Q&A]
I'm trying to get PWM on PIN_PD5 with TCD using AVR32DD14. I'm using version 1.5.11 of DxCore.

According to the documentation, since analogWrite is PORTMUX-aware,  this should be as easy as:

```
void setup() {
 // PORTMUX.TCDROUTEA = PORTMUX_TCD0_ALT4_gc;
}

void loop() {
  analogWrite(PIN_PD5, 128);
}
```

This gives me nothing on PD5. I've tried manually setting PORTMUX.TCDROUTEA to PORTMUX_TCD0_ALT4_gc and it has also no effect.

PWM on e.g. PIN_PC1 works just fine. Can someone confirm this before I file a bug report? Or am I missing something here?

**Comment by SpenceKonde** [comment]
```
void setup() {
 PORTMUX.TCDROUTEA = PORTMUX_TCD0_ALT4_gc;
}

void loop() {
  analogWrite(PIN_PD5, 128);
}
```

should work. Hopefully that's what you have.

**Reply by fijam** [reply]
It doesn't. I'm getting nothing on PD5.
